bin/start-local-node.sh

```diff
@@ -73,11 +77,11 @@ fi
 
 # Download drive from defined branch
 mn_bootstrap_drive_options="--drive-image-build-path="
-if [ -n "$drive_branch" ]
+if [ -n "$drive_branch" ] && [ -n "$drive_slug" ]
```

Comment:
I guess if did’t pass the slug it should use the default one. BTW, don’t `drive_repo` sounds better?

Reply:
OK, renamed the variable. I don't really understand what this tool does or how to test it without seeing it run in a Travis build. I tried to test locally but it wants a `package.json`, but the docs don't specify which package it tests. I think if either of the variables is null then it will just use the version installed by mn-bootstrap instead? Same as if `$drive_branch` or `$dapi_branch` are not set...